src/Page1.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+class Page1 extends React.Component {
+  render() {
```

Comment:
define the state here inside page1 component and defined function for changing the state

Reply:
https://github.com/RahafSalah/second-react-app/pull/4